kube/README.md

```diff
@@ -46,10 +44,12 @@ $ kubectlc exec -it m3nsch-client-78987775f5-l5pvs
 15:24:23.205737[I] workload started!
 ```
 
-You can view the stats of the ongoing benchmark via `m3nsch_server`'s prometheus endpoints:
+You can view the stats of the ongoing benchmark via `m3nsch_server`'s prometheus endpoints, or using our [grafana dashboard][dash]:
 ```
 $ kubectl port-forward svc/m3nch-agent 12580
 $ curl -sSf localhost:12580/metrics | rg write.+success
 # TYPE write_req_success counter
 write_req_success 146000
 ```
+
+[dash]: https://github.com/m3db/m3/blob/75661bc2e7fc806ee3424c068db6812455ba0878/integrations/grafana/m3nsch_dashboard.json
```

Comment:
Shouldn't this reference the master blob?  Otherwise they'll get an old dashboard if we ever update it no?

Reply:
I'm fine with making it master but I mostly wanted to future-proof this so the link still works if we ever move the dashboards around